data-prepper-plugins/peer-forwarder/src/test/java/com/amazon/dataprepper/plugins/processor/peerforwarder/PeerForwarderConfigTest.java

```diff
@@ -3,22 +3,51 @@
 import com.amazon.dataprepper.model.configuration.PluginSetting;
 import com.amazon.dataprepper.plugins.processor.peerforwarder.discovery.DiscoveryMode;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
 
+import java.io.File;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
 public class PeerForwarderConfigTest {
+    private static final String VALID_SSL_KEY_CERT_FILE = PeerForwarderConfigTest.class.getClassLoader()
+            .getResource("test-crt.crt").getFile();
+    private static final String INVALID_SSL_KEY_CERT_FILE = "";
+    private static List<String> TEST_ENDPOINTS = Arrays.asList("172.0.0.1", "172.0.0.2");
+
+    private static final MockedStatic<PeerClientPool> mockedPeerClientPoolClass = Mockito.mockStatic(PeerClientPool.class);
```

Comment:
This line might be causing problems - per [the MockedStatic docs](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/MockedStatic.html):
>The static mock is released when this object's ScopedMock.close() method is invoked. If this object is never closed, the static mock will remain active on the initiating thread. It is therefore recommended to create this object within a try-with-resources statement unless when managed explicitly, for example by using a JUnit rule or extension.